### PR TITLE
Update helm template example for secondary network and MacvlanNetwork

### LINKED ISSUE #46: [enhancement] Support MacVlan secondary network in helm
PR #37 introduced support for macvlan secondary network. We should enable it via helm.

### PR DESCRIPTION
**Pull request by Mmduh-483**
Fixes #46

### CODE REVIEW

File: manifests/stage-macvlan-cni/0010-macvlan-ds.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: macvlan-cni-ds
```
Comment (moshe010) [inline manifests/stage-macvlan-cni/0010-macvlan-ds.yml:17]: I don't think we want to maintain images per generic cni. 
Another option is just to install containernetworking-plugins

**Review by moshe010 — CHANGES_REQUESTED** [review]

File: manifests/stage-macvlan-cni/0010-macvlan-ds.yml (hunk 1 above)
Comment (adrianchiris) [reply manifests/stage-macvlan-cni/0010-macvlan-ds.yml:17]: This discussion should be on PR#37 IMO :)

Comment (adrianchiris) [reply manifests/stage-macvlan-cni/0010-macvlan-ds.yml:17]: So here is what i propose,

rename the stage to be : `stage-containernetworking-plugins` which will deploy a daemonset that will install (cp to /opt/cni/bin on host) the provided cni plugins.

it means we build one image for all plugins based on:
binaries published : https://github.com/containernetworking/plugins/releases

i prefer not to rely on distro specific repos

@moshe010 WDYT ?

File: deployment/network-operator/README.md [hunk 2]
```diff
@@ -122,6 +123,55 @@ resources:
...
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `ipamPlugin.deploy` | bool | `true` | Deploy IPAM CNI Plugin Secondary Network  |
+| `ipamPlugin.image` | string | `whereabouts` | IPAM CNI Plugin image name  |
+| `ipamPlugin.repository` | string | `dougbtv` | IPAM CNI Plugin image repository  |
+| `ipamPlugin.version` | string | `latest` | IPAM CNI Plugin image version  |
+
+### MacvlanNetwork Custom resource parameters
+
+| Name | Type | Default | description |
```
Comment (adrianchiris) [inline deployment/network-operator/README.md:167]: Im kinda leaning towards not having this part of the chart.
but rather have the user define after deployment.

maybe the user needs more than one ? or none ?

File: deployment/network-operator/README.md [hunk 3]
```diff
@@ -122,6 +122,45 @@ resources:
...
+
+##### Multus CNI Secondary Network
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `multus.deploy` | bool | `true` | Deploy Multus Secondary Network  |
+| `multus.image` | string | `multus` | Multus image name  |
+| `multus.repository` | string | `nfvpe` | Multus image repository  |
+| `multus.version` | string | `v3.6` | Multus image version  |
+| `multus.config` | string | `` | Multus CNI config if config is missing or empty then multus config will be automatically generated from the CNI configuration file of the master plugin (the first file in lexicographical order in cni-conf-dir)  |
```
Comment (adrianchiris) [inline deployment/network-operator/README.md:153]: > Multus CNI config if config is missing or empty then multus config will be automatically...

Multus CNI config, if empty then config will be automatically....

File: deployment/network-operator/README.md [hunk 4]
```diff
@@ -135,7 +174,7 @@ of the network operator in the following manner:
 
 #### Example 1
 Network Operator deployment with a specific version of OFED driver and a single RDMA resource mapped to `enp1`
-netdev.
+netdev, also deploying secondary network with Mutlus CNI, containernetworking-plugins CNI plugins and whereabouts IPAM CNI Plugin.
```
Comment (adrianchiris) [inline deployment/network-operator/README.md:177]: lets have this as example 3 which focuses on secondary network enablement with RDMA

i.e
deployment of device plugin and secondary network components.

device selectors PR can then be integrated into it to have a simple recipe to enable RDMA on secondary network.

**Review by adrianchiris — CHANGES_REQUESTED** [review]
small rework to the README and we are good to go !

File: deployment/network-operator/README.md [hunk 5]
```diff
@@ -169,3 +208,34 @@ devicePlugin:
     - name: rdma_shared_device_b
       devices: [ib0]
 ```
+
+#### Example 3
+Network Operator deployment with the default version of RDMA resources, second mapped to `ib0`,
```
Comment (adrianchiris) [inline deployment/network-operator/README.md:213]: i dont understand this sentence

**Review by adrianchiris — CHANGES_REQUESTED** [review]

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**